apps/api/migrations/novu-integrations/novu-integrations.migration.ts

```diff
@@ -0,0 +1,85 @@
+import '../../src/config';
+import {
+  OrganizationRepository,
+  EnvironmentRepository,
+  IntegrationRepository,
+  ChannelTypeEnum,
+  EnvironmentEntity,
+} from '@novu/dal';
+import { EmailProviderIdEnum, SmsProviderIdEnum } from '@novu/shared';
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from '../../src/app.module';
+
+const organizationRepository = new OrganizationRepository();
+const environmentRepository = new EnvironmentRepository();
+const integrationRepository = new IntegrationRepository();
+
+const createNovuIntegration = async (
+  environment: EnvironmentEntity,
+  channel: ChannelTypeEnum.EMAIL | ChannelTypeEnum.SMS
+) => {
+  const providerId = channel === ChannelTypeEnum.SMS ? SmsProviderIdEnum.Novu : EmailProviderIdEnum.Novu;
+
+  const count = await integrationRepository.count({
+    _environmentId: environment._id,
+    _organizationId: environment._organizationId,
+    providerId,
+    channel,
+  });
+
+  if (count > 0) {
+    return;
+  }
+
+  const countChannelIntegrations = await integrationRepository.count({
+    _environmentId: environment._id,
+    _organizationId: environment._organizationId,
+    channel,
+  });
```

Comment:
Would we want to skip the creation if the user already has integration on the same channel?

Reply:
No we just like to keep it disabled so the user that are used to see them still see them as usual even with the new list design